metomi/isodatetime/tests/test_00.py

```diff
@@ -894,6 +894,24 @@ def get_timerecurrence_membership_tests():
     ]
 
 
+def get_timerecurrence_first_after_tests():
+    """Return test get_first_after expressions for data.TimeRecurrence"""
+    return [
+        ("R5/2020-01-01T00:00:00Z/2020-01-05T00:00:00Z",
+         [("2019-01-01T00:00:00Z", "2020-01-01T00:00:00Z"),
+          ("2020-01-02T00:00:00Z", "2020-01-05T00:00:00Z"),
+          ("2020-01-15T00:00:00Z", "2020-01-17T00:00:00Z"),
+          ("2021-01-01T00:00:00Z", "None")]),
+        ("R3/P700D/1961-W01-1T06,5Z",
+         [("1950-01-01T00:00:00Z", "1957-W10-1T06,5Z"),
+          ("1958-01-01T00:00:00Z", "1959-02-02T06:30:00Z"),
+          ("2021-01-01T00:00:00Z", "None")]),
+        ("R/2000-01-01T00:00:00Z/P1Y",
+         [("2001-01-01T00:00:00Z", "2002-01-01T00:00:00Z"),
+          ("2002-01-01T00:00:00Z", "2003-01-01T00:00:00Z")])
```

Comment:
Test for the type error condition I mentioned above
```suggestion
          ("2002-01-01T00:00:00Z", "2003-01-01T00:00:00Z")]),
        ("R1/2000-01-01T00:00:00Z/P1Y",
         [("2000-01-01T00:00:00Z", "None")])
```